collector/src/main/java/com/bitmovin/analytics/utils/HttpClient.java

```diff
@@ -17,7 +17,7 @@
     public static final MediaType JSON
             = MediaType.parse("application/json; charset=utf-8");
     private static final String TAG = "HttpClient";
-    private final OkHttpClient client = new OkHttpClient();
+    private OkHttpClient client = null;
```

Comment:
we should make a note somewhere to discuss this with the player team and urge them to fix this. It's ok here for now but this smells like something that will bite us in the ass :D in the end.

Does it make sense to move the HTTP Client into a late-init singleton implementation in a separate class?